### PR TITLE
jquery-ui: Update theming related pages

### PR DESCRIPTION
**Pull request by jzaefferer**
Drop the api page and link to the correct api.jqueryui.com page instead, delete the bad write-a-theme page, collapse the theming folder and fix the order of the widget-factory pages.

### CODE REVIEW

File: page/jquery-ui/themeroller.md [hunk 1]
```diff
@@ -36,13 +34,4 @@ Once the theme has been downloaded and unzipped, you will see a folder named `th
 
 ThemeRoller generates a customized version of the jQuery UI CSS Framework for developing your own ThemeRoller-ready jQuery components. The classes generated by this framework are designed to accommodate common user interface design situations and include states, icons, and various helper classes as well.
 
-For information on developing with the jQuery UI CSS Framework, visit our [Theming API documentation](/jquery-ui/theming/api).
-
-### ThemeRoller Links
```
Comment (jzaefferer) [inline page/jquery-ui/themeroller.md]: These were all very much outdated.

File: page/jquery-ui/themeroller.md [hunk 2]
```diff
@@ -36,13 +34,4 @@ Once the theme has been downloaded and unzipped, you will see a folder named `th
...
-* [Tutorial: Develop Your Own jQuery "ThemeRoller-Ready" Components, Filament Group](http://www.filamentgroup.com/lab/developer_your_own_jquery_themeroller_ready_components/)
-* [Introducing ThemeRoller: Design & Download Custom Themes for jQuery UI, Filament Group](http://www.filamentgroup.com/lab/introducing_themeroller_design_download_custom_themes_for_jquery_ui/)
-* [Videos about Design & Download Custom Themes for jQuery UI, Filament Group](http://vimeo.com/1113981?pg=embed&sec=1113981)
-
-### Credits
```
Comment (jzaefferer) [inline page/jquery-ui/themeroller.md]: That applied to the initial version, the current version was almost completely rewritten.

**Comment by agcolom** [comment]
@scottgonzalez Could you please review this PR? It is related to UI. Thank you.

File: page/jquery-ui/theming.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 
-Since the framework styles only cover look and feel, plugin specific stylesheets are included that contain all the additional structural style rules required to make the widget functional, such as dimensions, padding, margins, positioning, and floats. Stylesheets for each plugin are located in the `themes/base` folder of the download and are named to match the plugin such as "jquery.ui.accordion.css". These styles must be carefully edited because they work in conjunction with the scripting and provide overrides of framework styles as needed.
+Since the framework styles only cover look and feel, plugin specific stylesheets are included that contain all the additional structural style rules required to make the widget functional, such as dimensions, padding, margins, positioning, and floats. Stylesheets for each plugin are located in the `themes/base` folder of the download and are named to match the plugin such as "accordion.css". These styles must be carefully edited because they work in conjunction with the scripting and provide overrides of framework styles as needed.
```
Comment (scottgonzalez) [inline page/jquery-ui/theming.md]: Individual files are no longer provided.

File: page/jquery-ui/theming.md [hunk 4]
```diff
@@ -17,8 +17,6 @@ We encourage all developers creating jQuery plugins to leverage the jQuery UI CS
...
-* **Modify the CSS files**: To get a bit more control over the look and feel, you may choose to start with the default theme (Smoothness) or a ThemeRoller-generated theme and then adjust the `ui.theme.css` file or any of the individual plugin stylesheets. For example, you could easily tweak the corner radius for all buttons to be different than the rest of the UI components or change the path for the icon sprite to use a custom set.  With a bit of style scoping, you can even use multiple themes together in a single UI. To keep maintenance simple, restricting changes to just the `ui.theme.css` file and images is recommended.
+* **Download a ThemeRoller theme**: The easiest way to build a theme is to use the [ThemeRoller](/jquery-ui/theming/themeroller/) to generate and download a theme. This app will create a new `theme.css` file and an `images` directory containing all necessary background images and icon sprites which can simply be dropped into your project. This approach will be the easiest to create and maintain but limits customization to the options provided in ThemeRoller.
```
Comment (scottgonzalez) [inline page/jquery-ui/theming.md]: "the ThemeRoller" -> "ThemeRoller"

Comment (scottgonzalez) [inline page/jquery-ui/theming.md]: `jquery-ui.theme.css`

File: page/jquery-ui/theming.md [hunk 5]
```diff
@@ -17,8 +17,6 @@ We encourage all developers creating jQuery plugins to leverage the jQuery UI CS
...
+* **Modify the CSS files**: To get a bit more control over the look and feel, you may choose to start with the default theme (Smoothness) or a ThemeRoller-generated theme and then adjust the `theme.css` file or any of the individual plugin stylesheets. For example, you could easily tweak the corner radius for all buttons to be different than the rest of the UI components or change the path for the icon sprite to use a custom set.  With a bit of style scoping, you can even use multiple themes together in a single UI. To keep maintenance simple, restricting changes to just the `theme.css` file and images is recommended.
```
Comment (scottgonzalez) [inline page/jquery-ui/theming.md]: More references to `theme.css`.

**Comment by jzaefferer** [comment]
Added a commit that fixes the ThemeRoller and theme.css references.

**Comment by scottgonzalez** [comment]
I didn't re-read the whole thing, but the changes look good now.

**Comment by jzaefferer** [comment]
Merged and tagged as 0.6.0.

**Pull request closed without merging**